tests/shim.js

```diff
@@ -0,0 +1,6 @@
+// This file is required to alleviate an RAF error that appears:
+// https://github.com/facebook/jest/issues/4545
+
+global.requestAnimationFrame = (callback) => {
```

Comment:
this is not needed; `airbnb-browser-shims` should provide all the shims we need including this one.

Reply:
Unfortunately, blanket importing `airbnb-browser-shims` causes a nebulous `parentNode is undefined` error from `matchmedia-polyfill`. Alternatively we could piecemeal import the `raf` polyfill on its own.

Reply:
hm, that seems like something we should fix - would you mind filing an issue on browser-shims for that?

Reply:
Yeppers

Reply:
https://github.com/airbnb/browser-shims/issues/20

Reply:
This is now resolved, by an update of matchmedia-polyfill.